xla/service/gpu/llvm_gpu_backend/BUILD

```diff
@@ -59,6 +59,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "ptx_version_util",
+    srcs = [
+        "ptx_version_util.cc",
+    ],
+    hdrs = [
+        "ptx_version_util.h",
+    ],
+    tags = [
+        "gpu",
+    ],
```

Comment:
Let's remove the tag. As far as I can tell, this library does not have backend specific dependencies and can be built for all backends.